server/plugin_test.go

```diff
@@ -220,13 +221,54 @@ func TestSpecialCases(t *testing.T) {
 
 	for _, tt := range tests {
 		t.Run(tt.inputMessage, func(t *testing.T) {
-			post := &model.Post{
-				Message: tt.inputMessage,
+			{
```

Comment:
LOL, that's an interesting way of formatting! 0/5 a slight preference to make each "block" a separate t.Run, and use the test names for comments - makes the logs pretty?

Reply:
Yeah I don't have a lot of experience structuring go tests, but I also felt the whole test could probably be re-arranged to more clearly report which test failed. I decided that would be better tackled by others...

Reply:
Ok, I'll do it later.

Reply:
BTW in case it's not obvious, the separate blocks are there to ensure we destroy the `post` instances because the plugin mutates the ones passed in... and it seemed even more messy to declare them all with separate names.